cmd/mdatagen/internal/samplereceiver/metadata_custom.yaml

```diff
@@ -0,0 +1,201 @@
+# Sample metadata file with all available configurations for a receiver.
+
+type: sample
+scope_name: go.opentelemetry.io/collector/internal/receiver/samplereceiver
+github_project: open-telemetry/opentelemetry-collector
+
+generated_package_name: custom
```

Comment:
`internal/metadata` package is abandoned now and should be removed, right?

Reply:
_Commented too early under assumption that https://github.com/open-telemetry/opentelemetry-collector/pull/11468/files#diff-7d3195204f99934e80e03730709378a3054a1c056d953091368ad2232d282442R5 line is removed_